app/components/upload-app/status/index.hbs

```diff
@@ -0,0 +1,126 @@
+
+    <AkLoader
+      data-test-uploadAppStatus-loader
+      @size={{30}}
+      @thickness='5'
+      local-class='{{this.submissionNumbers.loaderColor}} cursor-pointer'
+      @progress='100'
+      @variant={{this.submissionNumbers.variant}}
+      {{did-insert this.registerAnchorRef}}
+    >
+      <:label>
+        <AkIcon
+          data-test-uploadAppStatus-icon
+          @iconName='file-upload'
+          @variant='outlined'
+          @size='small'
+        />
+      </:label>
+    </AkLoader>
+  {{/if}}
+</AkStack>
+
+{{#if (or this.hasSortedSubmissions this.uploadedApps.files.length)}}
+  <AttachPopover
+    @interactive={{true}}
+    @arrow={{true}}
+    @placement='bottom'
+    @lazyRender={{true}}
+    @showOn='click'
+    @hideOn='clickout click'
+    @animation='shift'
+    @class='ember-attacher-popper {{local-class "popper"}}'
+    @popperTarget={{this.anchorRef}}
+  >
+    <AkStack
+      @width='full'
+      @direction='column'
+      local-class='notification-dropdown-container'
+      data-test-uploadAppStatusPopover-container
+    >
+      <AkStack @width='full' @justifyContent='space-between' class='p-2'>
+        <AkStack @spacing='1'>
+          <AkIcon @iconName='show-chart' />
+
+          <AkTypography
+            data-test-uploadAppStatusPopover-title
+            @variant='subtitle1'
+          >
+            {{t 'uploadStatus'}}
+          </AkTypography>
+        </AkStack>
+
+        <AkStack @spacing='2'>
+          <AkStack @alignItems='center'>
+            <AkIcon
+              @iconName='downloading'
+              @size='small'
+              local-class='download-icon-color'
+            />
+
+            <AkTypography
+              data-test-uploadAppStatus-submissionCount='running'
+              @variant='body2'
+              local-class='ml-half'
+            >
+              0{{this.submissionNumbers.runningSubmissions}}
+            </AkTypography>
+          </AkStack>
+
+          <AkStack @alignItems='center'>
+            <AkIcon @iconName='download-done' @color='success' @size='small' />
+
+            <AkTypography
+              data-test-uploadAppStatus-submissionCount='completed'
+              @variant='body2'
+              local-class='ml-half'
+            >
+              0{{this.submissionNumbers.completedSubmissions}}
+            </AkTypography>
+          </AkStack>
+
+          <AkStack @alignItems='center'>
+            <AkIcon @iconName='error' @color='error' @size='small' />
+
+            <AkTypography
+              data-test-uploadAppStatus-submissionCount='failed'
+              @variant='body2'
+              local-class='ml-half'
+            >
+              0{{this.submissionNumbers.failedSubmissions}}
+            </AkTypography>
+          </AkStack>
+        </AkStack>
+      </AkStack>
+
+      <AkDivider />
+
+      {{#if (or this.hasSubmissions this.uploadedApps.files.length)}}
+        {{#if (or this.hasSortedSubmissions this.uploadedApps.files.length)}}
```

Comment:
this check can be done through a getter to simplify
`return (this.hasSubmissions || this.uploadedApps.files.length) || this.hasSortedSubmissions`